README.md

```
@@ -156,42 +165,287 @@ The system implements 8 distinct roles to ensure strict access control:
+# Start the backend server with auto-reload
+npm run dev
+
+# Or for production mode
 npm start
+```
+
+The server will start on `http://localhost:3000`
+
+### 7. Access the Application
+Open your browser and navigate to:
+- **Main Application**: http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)